src/components/FeaturedProducts.js

```diff
@@ -0,0 +1,59 @@
+
+import React from 'react'
+import featuredProducts from '../data/featured-products.json'
+import './Home.css'
+
+function FeaturedProducts(){
+    const products = featuredProducts.results;
+    const [productItems] = React.useState(products);
+    /*
+        the main image of the product, its name, category, and price.
+        image: obj.data.mainimage.url (has dimensions 696 900)
+        name:  obj.data.name
+        category: obj.data.category.slug
+        price: obj.data.price
+    */
+
+    const GridItem = (props) => {
+        return (
+        <div key={props.name} className="grid-item" >
+            <div className="grid-image">
+                <img className="product-image" src={props.image} alt={props.name}/>
+            </div>
+            <div>
+                Name:<span>{props.name}</span>
+            </div>
+            <div>
+                Category:<span>{props.category}</span>
+            </div>
+            <div>
+                Price:<span style={{color:"green"}}>${props.price}</span>
+            </div>
+            
+        </div>);
+    }
```

Comment:
Here you can also deconstruct the props by only receiving the props that you are using like category, image...
And with that you dont need to use 'props.' in each variable

Also, question for reflexion... always the name will be unique? If not probably we need to change a little bit the key
```suggestion
    const GridItem = ({category, image, name, price}) => {
        return (
        <div key={name} className="grid-item" >
            <div className="grid-image">
                <img className="product-image" src={image} alt={name}/>
            </div>
            <div>
                Name:<span>{name}</span>
            </div>
            <div>
                Category:<span>{category}</span>
            </div>
            <div>
                Price:<span style={{color:"green"}}>${price}</span>
            </div>
            
        </div>);
    }
```